editor/editor_help_search.cpp

```diff
@@ -575,7 +586,7 @@ TreeItem *EditorHelpSearch::Runner::_create_class_item(TreeItem *p_parent, const
 	return item;
 }
 
-TreeItem *EditorHelpSearch::Runner::_create_method_item(TreeItem *p_parent, const DocData::ClassDoc *p_class_doc, const String &p_text, const DocData::MethodDoc *p_doc) {
+TreeItem *EditorHelpSearch::Runner::_create_method_item(TreeItem *p_parent, const DocData::ClassDoc *p_class_doc, const String &p_text, const DocData::MethodDoc *p_doc, bool p_is_annotation) {
```

Comment:
I would create a dedicated method instead. Signal also uses MethodDoc and is formatted similarly, yet it has a dedicated method. A little code duplication doesn't hurt here (and it's better to instead factor out similar bits of unique methods to a dedicated helper, if this gets too obnoxious).